### ISSUE TITLE
Error when trying to enable plugin

### ISSUE THREAD

**Issue by Lamby777** [dandeliondino/tile_bit_tools#58]
On Godot v4.0 stable mono, plugin installed via asset library tab.

```
Unable to load addon script from path: 'res://addons/tile_bit_tools/plugin.gd'. This might be due to a code error in that script.
Disabling the addon at 'res://addons/tile_bit_tools/plugin.cfg' to prevent further errors.
```

Checked the `plugin.gd` and yeah, seems to be an error accessing something that doesn't exist.
![image](https://github.com/dandeliondino/tile_bit_tools/assets/35111165/e3644623-80c7-4b93-a18b-cfd7ad751818)

**Comment by dandeliondino** [comment]
Thank you for the report! I believe it's due to inadvertently shadowing the Globals class. Please let me know if it's fixed for you in 1.0.3.

**Comment by Lamby777** [comment]
That was FAST!

Works! TY
![image](https://github.com/dandeliondino/tile_bit_tools/assets/35111165/3d09b1b3-f040-4289-a0b2-e6760f2d2365)
